### ISSUE TITLE
Cargo workspaces not supported

### ISSUE THREAD

**Issue by ruabmbua** [rustwasm/wasm-pack#468]
## 🐛 Bug description
wasm-pack test works with Cargo workspaces, if I specify the path to my to be tested lib, but wasm-pack build throws an error, that `target/wasm32-unknown-unknown/release/wasm_module.wasm` can not be found.

**Part of command output:**
```
  [8/9] wasm-bindgen already installed...
  [9/9] Running WASM-bindgen...
  error: failed to read `target/wasm32-unknown-unknown/release/wasm_module.wasm`
| 	caused by: No such file or directory (os error 2)
Running the wasm-bindgen CLI
Caused by: Process exited with exit code: 1: `wasm-bindgen` did not exit successfully.

stdout:

stderr:

error: failed to read `target/wasm32-unknown-unknown/release/wasm_module.wasm`
	caused by: No such file or directory (os error 2)

```

#### 🤔 Expected Behavior
Both commands should work.

#### 👟 Steps to reproduce

* Create a Cargo workspace, and add a project derived from the project template to it.
* Run wasm-pack build <path/to/your/crate> in the workspace root directory.

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.5.1
rustc version: rustc 1.31.1 (b6c32da9b 2018-12-18)

**Comment by ishitatsuyuki** [comment]
Duplicate of #252

**Comment by ashleygwilliams** [comment]
hi! current master, which will become 0.6.0 *soon*, has a fix for this. you can install from master by running `cargo uninstall wasm-pack && cargo install wasm-pack --git https://github.com/rustwasm/wasm-pack/`. will close as this is fixed, and just waiting on release.
